### ISSUE TITLE
EventGet() always results in error

### ISSUE THREAD

**Issue by Gys** [apzuk/go-eventbrite#5]
Hi

As a first try, I created a few lines of code to get info on an existing event. 

	event, err := clnt.EventGet(context.Background(), "<event>")

This seems to result always in an error: `validator: (nil url.Values)`
Because EventGet() effectively calls `return result, c.getJSON(ctx, "/events/"+id, url.Values{}, &result)`

And c.getJSON() calls c.get().

Inside the get() is a validator for url.Values{} (passed as apiReq):

	if apiReq != nil {
		if err := validate.Struct(apiReq); err != nil {
			return nil, err
		}
	}

 The empty value of url.Values{} is `map[]` and not nil, so it is validated, but an empty map is invalid. Hence the error.

If I disable the validator in get() then the EventGet() works.

**Comment by fawkesley** [comment]
I've just hit this too, thanks @Gys for raising this. Did you carry on with a forked version?

**Comment by Gys** [comment]
No, I created my own local version because I needed some other changes. Sorry.

**Comment by fawkesley** [comment]
FWIW as a quick patch I got this working by replacing `url.Values{}` [inside `EventGet`](https://github.com/apzuk3/go-eventbrite/blob/v3/event.go#L540) with `nil`, e.g.:

before:
```
return result, c.getJSON(ctx, "/events/"+id, url.Values{}, &result)
```

after:
```
result, c.getJSON(ctx, "/events/"+id, nil, &result)
```

@Gys No worries, was this the patch you applied too?

**Comment by Gys** [comment]
getJSON() is called from many places?
Had a quick look and think I simply disabled the validation code inside Get():

```
if apiReq != nil {
	if err := validate.Struct(apiReq); err != nil {
		return nil, err
	}
}

```
